CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Changelog
 
 ## main
-
 * Add --prefix option to load-models
 * Add get-model-source command
+* Use models path as a default prefix (could be overwitten by --prefix flag)
```

Comment:
what *exactly* is the semantic for use of paths in file names? aside from the prefix arg? eg: i assume we canonicalize paths in some way? what does ../foo/../bar/baz/model.rel do? what if its a path from home, eg: ~/bar/baz/model? and what about a fully qualified path? the original idea was to use the basename, and prepend any prefix provided by the prefix arg. simple and unambiguous. what use case does this not handle?